src/webcomponents/clinical/obsolete/opencga-clinical-analysis-writer.js

```diff
@@ -541,10 +541,14 @@ export default class OpencgaClinicalAnalysisWriter extends LitElement {
                             type: "custom",
                             display: {
                                 render: data => {
-                                    return html`<individual-id-autocomplete .opencgaSession="${this.opencgaSession}" .config=${{
-                                        addButton: false,
-                                        multiple: false
-                                    }} @filterChange="${e => this.onCancerChange(e)}"></individual-id-autocomplete>`;
+                                    return html`<individual-id-autocomplete
+                                        .opencgaSession="${this.opencgaSession}"
+                                        .config=${{
+                                            addButton: false,
+                                            multiple: false
+                                        }}
+                                        @filterChange="${e => this.onCancerChange(e)}">
+                                    </individual-id-autocomplete>`;
```

Comment:
@antonioaltamura this component is obsolete and is not used by any other component in IVA.